spec/models/extract_fetcher_spec.rb

```diff
@@ -119,4 +119,17 @@
       expect(fetch.augment_subject_ids([s1.id])).to contain_exactly(s1.id, s2.id)
     end
   end
+
+  it 'skips getting user extracts if user id is nil' do
+    create :extract, workflow: wf, user_id: user1_id
+    nil_extract = create :extract, workflow: wf, user_id: nil
+
+    allow(Extract).to receive(:where).and_return([nil_extract])
```

Comment:
Not sure you need to allow the `.where` method on the AR model if it is never received on line 132.

Reply:
I was puzzled by this as well, but without the `allow` rspec then complains that the method is neither a mock nor a spy